utoipa-swagger-ui/src/lib.rs

```diff
@@ -539,6 +543,10 @@ pub struct Config<'a> {
     /// [`oauth::Config`] the Swagger UI is using for auth flow.
     #[serde(skip)]
     oauth: Option<oauth::Config>,
+
+    /// [ layout ] the layout of Swagger UI uses, default is "StandaloneLayout"
+    #[serde(default = "swagger_default_layout")]
+    layout: String,
```

Comment:
If possible the layout will live at least as long as the config does.
```suggestion
    layout: &'a str,
```